nextjs-app-v14/src/app/components/button/page.tsx

```diff
@@ -1,23 +1,10 @@
-'use client';
+import  Button  from './button';
+import { Metadata } from 'next'
 
-import NavigationLayout from "@/layout/navigation";
-import { PieButton } from '@justeattakeaway/pie-webc/react/button.js';
-import { PieDivider } from '@justeattakeaway/pie-webc/react/divider.js';
-import { IconSearch } from '@justeattakeaway/pie-icons-webc/dist/react/IconSearch.js';
-import { useState } from "react";
-
-export default function Button() {
-    const [counter, setCounter] = useState(0);
+export const metadata: Metadata = {
+    title: 'Button',
+}
 
-    return (
-        <NavigationLayout title="Button">
-        <PieButton onClick={() => setCounter(counter + 1)}>Counter: {counter}</PieButton>
-        <PieDivider />
-        <h2>PIE Button - with Icon and text</h2>
-        <PieButton>
-            <IconSearch slot="icon"></IconSearch>
-            Search
-        </PieButton>
-        </NavigationLayout>
-    );
+export default function ButtonComponent() {
+    return <Button/>;
```

Comment:
i would just import the components here directly instead of wrapping them. wdyt?

Reply:
Same as above.